include/detect_breakends/junction_detection.hpp

```diff
@@ -15,10 +15,10 @@
  *
  * \param alignment_file_path input file - path to the sam/bam file
```

Comment:
The usage of the `-` in the documentation comments should be made consistent. Some params have the `-` after the parameter name, others in the middle of the description and others do not have any `-`.
My suggestion:
`\param parameter_name - parameter description`